service/controller/v3/resource/vnetpeeringcleaner/create.go

```diff
@@ -12,21 +13,47 @@ import (
 // EnsureCreated ensure that vnetpeering resource are deleted,
 // since they are no longer in use in this version.
 func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
-	r.logger.LogCtx(ctx, "level", "debug", "message", "ensuring deletion of host vnetpeering")
+	r.logger.LogCtx(ctx, "level", "debug", "message", "ensuring deletion of vnetpeerings")
 
 	azureConfig, err := key.ToCustomObject(obj)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
-	vnetPeeringClient, err := r.getVnetPeeringClient()
+	vnetPeeringHostClient, err := r.getVnetPeeringHostClient()
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
 	resourceGroupName := r.azure.HostCluster.ResourceGroup
 	vnetName := r.azure.HostCluster.ResourceGroup
 	peeringName := key.ResourceGroupName(azureConfig)
+	err = deletePeering(ctx, vnetPeeringHostClient, resourceGroupName, vnetName, peeringName)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", "ensured delete host vnetpeering")
```

Comment:
The wording of the logs should be aligned with above.

Reply:
done

Reply:
This is nitpick but it should be `ensured deletion of`